.github/labeler.yml

```diff
@@ -0,0 +1,14 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# Configuration for auto-labeling PRs
+#
+# Note that we enable dot, so "**" matches all files in a folder
+
+Z-BenchCI:
+  - kani-compiler/**
+  - rust-toolchain.toml
+  - kani-dependencies
+  - kani-driver/src/call-*
+  - Cargo.lock
+
```

Comment:
I think we should add `cprover_bindings/**` as well.

Reply:
Perhaps also `library/**`?

Reply:
Sure. I thought about adding library, but I'm assuming most changes are new APIs or documentation. Anyway, we can manually add or remove the label if we feel like it's not needed